src/main/java/net/mcreator/ui/modgui/ArmorGUI.java

```diff
@@ -167,10 +167,10 @@ public class ArmorGUI extends ModElementGUI<Armor> {
 
 	private final SoundSelector equipSound = new SoundSelector(mcreator);
 
-	private final JSpinner maxDamage = new JSpinner(new SpinnerNumberModel(25, 0, 1024, 1));
+	private final JSpinner maxDamage = new JSpinner(new SpinnerNumberModel(15, 0, 1024, 1));
 	private final JSpinner damageValueBoots = new JSpinner(new SpinnerNumberModel(2, 0, 1024, 1));
-	private final JSpinner damageValueLeggings = new JSpinner(new SpinnerNumberModel(5, 0, 1024, 1));
-	private final JSpinner damageValueBody = new JSpinner(new SpinnerNumberModel(6, 0, 1024, 1));
+	private final JSpinner damageValueLeggings = new JSpinner(new SpinnerNumberModel(6, 0, 1024, 1));
```

Comment:
Order is indeed reversed here so revert this back as noted in https://github.com/MCreator/MCreator/pull/4619#issuecomment-1937083627

Reply:
Ok

Reply:
Like this?
https://github.com/MCreator/MCreator/pull/4619/commits/034efab9ebad78ad2eff7ef876f5387e7b829ccd

Reply:
Yep :)